### PR TITLE
remove old important message from reports API

### PR DESCRIPTION
**Pull request by kiangol**
- [ ] [API update message](https://github.com/bring/developer-site#tips-for-writing-an-api-update-message) (changelog entry) has been reviewed by Mybring Experience.

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/6k3XSXsh](https://trello.com/c/6k3XSXsh)

File: content/api/reports/_index.md [hunk 1]
```diff
@@ -11,12 +11,6 @@ menu:
 weight: 51
 
 important:
-  - type: warn
```
Comment (mbrameshkumar) [inline content/api/reports/_index.md:14]: Do we have to update version history for this removal of info message from developer page 🤔

Comment (kiangol) [reply content/api/reports/_index.md:14]: I don't think so, since the API itself has not changed 🤔

Comment (mbrameshkumar) [reply content/api/reports/_index.md:14]: 👍

**Review by mbrameshkumar — APPROVED** [review]
👍

**Pull request merged**